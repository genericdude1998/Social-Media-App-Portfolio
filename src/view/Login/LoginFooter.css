.baseFooter{
    text-align: center;
    color: black;
}