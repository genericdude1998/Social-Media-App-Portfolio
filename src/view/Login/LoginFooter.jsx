import React from 'react';
import styles from './LoginFooter.css'

const LoginFooter = () => {
    return (
        <div className={styles.baseFooter}>
            Please type Admin as username and 1234 as password to begin!
        </div>
    );
}

export default LoginFooter;
