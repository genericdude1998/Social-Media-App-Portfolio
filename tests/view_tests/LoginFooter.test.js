import React from 'react';
import { shallow } from 'enzyme';
import LoginFooter from '../../src/view/Login/LoginFooter';

let loginFooterWrapper = shallow(
    <LoginFooter/>
);
it('should match the snapshot', () => {
    expect(loginFooterWrapper).toMatchSnapshot();
});