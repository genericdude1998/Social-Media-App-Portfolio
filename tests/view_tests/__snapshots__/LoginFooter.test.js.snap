// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="baseFooter"
>
  Please type Admin as username and 1234 as password to begin!
</div>
`;
